docs/business.yaml

```diff
@@ -503,6 +503,196 @@ paths:
                         waiveFees: false
+                change-of-directors-success-response:
+                  summary: Change Of Directors Response
+                  value:
+                    filing:
+                      business:
+                        foundingDate: '2024-07-08T07:00:00.000+00:00'
+                        identifier: BC0871442
+                        legalName: JIA TEST BC TO ULC
+                        legalType: ULC
+                      changeOfDirectors:
+                        directors:
+                          - actions:
+                            - appointed
+                            appointmentDate: '2024-07-09'
+                            cessationDate: null
+                            deliveryAddress:
+                              addressCity: Richmond
+                              addressCountry: CA
+                              addressRegion: BC
+                              deliveryInstructions: ''
+                              postalCode: V7E 1V2
+                              streetAddress: 10300 4th Ave
+                              streetAddressAdditional: ''
+                            mailingAddress:
+                              addressCity: Richmond
+                              addressCountry: CA
+                              addressRegion: BC
+                              deliveryInstructions: ''
+                              postalCode: V7E 1V2
+                              streetAddress: 10300 4th Ave
+                              streetAddressAdditional: ''
+                            officer:
+                              firstName: Doe
+                              lastName: Smith
+                              middleInitial: ''
+                              prevFirstName: ''
+                              prevLastName: ''
+                              prevMiddleInitial: ''
+                          - actions:
+                            - appointed
+                            appointmentDate: '2024-07-09'
+                            cessationDate: null
+                            deliveryAddress:
+                              addressCity: Victoria
+                              addressCountry: CA
+                              addressRegion: BC
+                              deliveryInstructions: ''
+                              postalCode: V8W 1W4
+                              streetAddress: 1-940 Blanshard
+                              streetAddressAdditional: ''
+                            mailingAddress:
+                              addressCity: Victoria
+                              addressCountry: CA
+                              addressRegion: BC
+                              deliveryInstructions: ''
+                              postalCode: V8W 1W4
+                              streetAddress: 1-940 Blanshard
+                              streetAddressAdditional: ''
+                            officer:
```

Comment:
I saw `partyType` in some of the `officer` object in the Postman collection, is it just optional?

Reply:
Thank you for pointing that out. 

Not sure about the `partyType`. According to the [business schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/directors.json), I did not see this field referenced in the person structure under change of directors. Currently, when sending a POST request without this `partyType`, the change of directors filing works correctly.

Should we discuss this situation like finding the reference where partyType is used? And think about how to deal with the case like if it needs to be included or if it's optional.

Reply:
as per our discussion, just update the Postman collection to follow the updated ones in the specs when you have time.